src/SaveObject.ts

```diff
@@ -750,7 +819,21 @@ async function loadGame(saveData: SaveData): Promise<boolean> {
   if (!saveData) return false;
   const jsonSaveString = await decodeSaveData(saveData);
 
-  const saveObj = JSON.parse(jsonSaveString, Reviver);
+  const saveObj = JSON.parse(jsonSaveString, Reviver) as {
```

Comment:
Same argument as before, but even moreso.

The current code is treating these as string, but we can't actually guarantee that. Any lint errors here are almost certainly valid.